### PR TITLE
CSHARP-3369: Adding solution filters

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

File: CSharpDriver.Driver.slnf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "solution": {
+    "path": "CSharpDriver.sln",
+    "projects": [
+      "src\\MongoDB.Bson\\MongoDB.Bson.csproj",
+      "src\\MongoDB.Driver.Core\\MongoDB.Driver.Core.csproj",
+      "src\\MongoDB.Driver\\MongoDB.Driver.csproj",
+      "tests\\MongoDB.Driver.TestHelpers\\MongoDB.Driver.TestHelpers.csproj",
```
Comment (DmitryLukyanov) [inline CSharpDriver.Driver.slnf:8]: if I'm not mistaken, we also should add this line `MongoDB.Driver.Core.TestHelpers` here

File: CSharpDriver.Driver.Core.slnf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "solution": {
+    "path": "CSharpDriver.sln",
+    "projects": [
+      "src\\MongoDB.Bson\\MongoDB.Bson.csproj",
+      "src\\MongoDB.Driver.Core\\MongoDB.Driver.Core.csproj",
```
Comment (rstam) [inline CSharpDriver.Driver.Core.slnf:6]: Add:

    "tests\\MongoDB.Bson.TestHelpers\\MongoDB.Bson.TestHelpers.csproj"

File: CSharpDriver.Driver.slnf (hunk 1 above)
Comment (rstam) [reply CSharpDriver.Driver.slnf:8]: Also: 

    "tests\\MongoDB.Bson.TestHelpers\\MongoDB.Bson.TestHelpers.csproj"

**Review by rstam — CHANGES_REQUESTED** [review]

File: CSharpDriver.Driver.Core.slnf [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+      "tests\\MongoDB.Driver.Core.TestHelpers\\MongoDB.Driver.Core.TestHelpers.csproj",
+      "tests\\MongoDB.Driver.Core.Tests\\MongoDB.Driver.Core.Tests.csproj"
+    ]
+  }
+}
```
Comment (DmitryLukyanov) [inline CSharpDriver.Driver.Core.slnf:11]: the last line should be empty, then you won't see this red circle

File: CSharpDriver.Driver.slnf [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "solution": {
+    "path": "CSharpDriver.sln",
+    "projects": [
+      "src\\MongoDB.Bson\\MongoDB.Bson.csproj",
+      "src\\MongoDB.Driver.Core\\MongoDB.Driver.Core.csproj",
+      "src\\MongoDB.Driver.GridFS\\MongoDB.Driver.GridFS.csproj",
```
Comment (DmitryLukyanov) [inline CSharpDriver.Driver.slnf:7]: nit: this is probably unnecessary since this lib references Core and Driver

**Review by DmitryLukyanov — APPROVED** [review]
minor nit and then LGTM

File: CSharpDriver.Driver.slnf (hunk 4 above)
Comment (rstam) [reply CSharpDriver.Driver.slnf:7]: I agree.

But also... I don't think we need to create yet another filter for the GridFS project. That should be used very rarely.

**Review by rstam — APPROVED** [review]
LGTM after you apply Dmitry's suggestion to remove GridFS

File: CSharpDriver.Driver.slnf (hunk 4 above)
Comment (DmitryLukyanov) [reply CSharpDriver.Driver.slnf:7]: Yeah, I'm just saying that this library is useless in this list since the related tests are not added

Comment (rstam) [reply CSharpDriver.Driver.slnf:7]: And I'm agreeing with you... we should remove this project from this filter.

Comment (BorisDog) [reply CSharpDriver.Driver.slnf:7]: Needed in MongoDB.Driver.Tests

**Comment by BorisDog** [comment]
Merged.

**Pull request closed without merging**